config/routes.rb

```diff
@@ -11,6 +11,11 @@
     end
     resources :comments, only: :create
   end
+  resources :users do
+    member do
+      post :follow
```

Comment:
Does this actually work?  I think you need to add work in the users controller too, unless I'm mistaken.